app/controllers/spree/admin/orders_controller.rb

```diff
@@ -38,6 +38,11 @@ def update
         @order.recreate_all_fees!
 
         unless order_params.present? && @order.update(order_params) && @order.line_items.present?
```

Comment:
We could reduce the 10 lines inside this block down to 6. What do you think?
```ruby
unless order_params.present? && @order.update(order_params) && @order.line_items.present?
  if @order.line_items.empty? && !params[:suppress_error_msg]
    @order.errors.add(:line_items, Spree.t('errors.messages.blank'))
  end

  flash[:error] = @order.errors.full_messages.join(', ') if @order.errors.present?
  return redirect_to spree.edit_admin_order_path(@order)
end
```

Reply:
I like this better as well, it avoids comparing a param to the string `"true"`.

Reply:
I agree. I'm not familiar with `unless` and ruby syntax (like the `if` at the end of the statement) and I don't have the reflexes . Will do the changes, thanks a lot.